src/components/formItem/formItem.svelte

```diff
@@ -135,10 +134,12 @@
 
       & .container:has(*:focus-visible),
       &.isFocused .container {
-        color: var(--color-focus);
-        background: var(--background-focus);
-        box-shadow: var(--shadow-focus);
-        border-color: var(--border-color-focus);
+        &:not(:has(.extra-content:focus-within)) {
```

Comment:
What's going on here?

Reply:
Only show the focus border if the focus is inside the input element, not within the .extra content element.